### PR TITLE
5700 404 message

### PR DESCRIPTION
**Pull request by wkhunt**

### CODE REVIEW

File: workflow/AlertDB.go [hunk 1]
```diff
@@ -117,27 +117,23 @@ func (db *AlertDB) GetAllByTrigger(format *piazza.JsonPagination, triggerId stri
...
-		return nil, LoggedError("AlertDB.GetOne failed: %s", err)
+		return nil, getResult.Found, LoggedError("AlertDB.GetOne failed: %s", err)
 	}
 	if getResult == nil {
-		return nil, LoggedError("AlertDB.GetOne failed: no getResult")
-	}
-
-	if !getResult.Found {
-		return nil, nil
+		return nil, true, LoggedError("AlertDB.GetOne failed: no getResult")
```
Comment (johnmcmahon) [inline workflow/AlertDB.go:126]: If getResult is nil, shouldn't this return false (not found)?  Maybe I don't understand the intent behind the bool

Comment (johnmcmahon) [inline workflow/AlertDB.go:126]: Ohhhhhh, if getResult is nil, that means it failed for a different reason, and we shouldn't send a 404?

Comment (wkhunt) [inline workflow/AlertDB.go:126]: the bool is `found`. This is the switch that determines whether or not an error is a 404 or a 400. If !Found, there will always be an error with the implementation of GetByID. I think it is actually impossible for getResult to be nil, just kept it there to be on the safe side

File: workflow/WorkflowService.go [hunk 2]
```diff
@@ -306,12 +306,12 @@ func (service *WorkflowService) GetAdminStats() *piazza.JsonResponse {
 // GetEventType TODO
 func (service *WorkflowService) GetEventType(id piazza.Ident) *piazza.JsonResponse {
 
-	event, err := service.eventTypeDB.GetOne(piazza.Ident(id))
-	if err != nil {
-		return service.statusNotFound(id)
+	event, found, err := service.eventTypeDB.GetOne(piazza.Ident(id))
+	if !found {
+		return service.statusNotFound(err)
```
Comment (johnmcmahon) [inline workflow/WorkflowService.go:311]: Does the err include the id that couldn't be found?

Comment (wkhunt) [inline workflow/WorkflowService.go:311]: Yes, in GetByID in index.go in pz-gocommon/elasticsearch if esi.ItemExists == false, then it returns a GetResult with Found = false and an error  that includes the es id, index, and type

Comment (johnmcmahon) [inline workflow/WorkflowService.go:311]: sounds good then

**Pull request merged**